generate_go.sh

```diff
@@ -15,8 +15,12 @@ for file in `ls *.proto`
 done
 
 echo "generate go code..."
+ret=0
 for file in `ls *.proto`  
     do
     base_name=$(basename $file ".proto")
     protoc --go_out=$GO_OUT_M:../pkg/$base_name $file
-done
\ No newline at end of file
+    last_ret=$?
```

Comment:
Good idea. I think `|| ret = $?` is enough. I still wanna protoc each file, but if some failed just exit no-zero.